frontend/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bazarr",
-  "version": "1",
+  "version": "1.0.0",
```

Comment:
Why do you change this file? It's not required.

Reply:
When running react-script build ti was returning an error because the version wasn't pasreable by semver

Reply:
@LASER-Yi what do you think about this one?

Reply:
It’s fine :)